src/osip-model/src/main/java/edu/kit/pse/osip/core/model/base/ProductionSite.java

```diff
@@ -83,20 +83,19 @@ private void initTanks() {
                 TankSelector.MIX.getInitialTemperature(), TankSelector.MIX.getInitialColor()),
                 new Pipe(SimulationConstants.PIPE_CROSSSECTION, SimulationConstants.PIPE_LENGTH, (byte) 100));
 
-        boolean specialPipeAssigned = false;  /* One of the pipes needs to be set to 34 instead of 33 */
+        byte count = 0;
         for (TankSelector selector: TankSelector.valuesWithoutMix()) {
             byte threshold;
-            if (!specialPipeAssigned) {
-                threshold = (byte) 34;
-                specialPipeAssigned = true;
+            if (count == 0 || count == 1) {
+                threshold = (byte) 50;
+                count++;
```

Comment:
Count++ würde ich in jedem Durchlauf zählen. Sollte auch so funktionieren, aber ein count, der dann irgendwann aufhört, ist seltsam